app/controllers/user/mygardens_controller.rb

```diff
@@ -1,6 +1,6 @@
 class User::MygardensController < User::BaseController 
 	def index
-		@gardens = current_user.gardens
+		@gardens = User.find(current_user.id).gardens
```

Comment:
were you running into problems with the way we were accessing the gardens before?

Reply:
i needs to access the database in order to be truly up to date, especially for the purposes of deleting